scipy/optimize/_optimize.py

```diff
@@ -1221,6 +1221,9 @@ def fmin_bfgs(f, x0, fprime=None, args=(), gtol=1e-5, norm=Inf,
         Print convergence message if True.
     retall : bool, optional
         Return a list of results at each iteration if True.
+    xtol : float
```

Comment:
Check with other minimisers first, but I think for consistency this should probably be `xrtol` because it's a relative change.

Reply:
Got it, I'll double check. Thanks @andyfaff.

Reply:
Looks like `xtol` in `_minimize_newtoncg` is relative as well, as a quick example, if I'm understanding what you meant correctly.

![image](https://user-images.githubusercontent.com/42592742/201553421-af804b4a-c695-4d01-86ec-cf4294f149c4.png)

Reply:
You could imagine there could be an absolute or relative stopping criterion for a minimiser. So fatol, frtol, xatol, xrtol, etc. Historically people used e.g. xtol but you don't know whether that's relative or absolute, and what happens when you want to implement the other?
This PR looks like implementing a relative tolerance for X, so instead of using xtol you need to use xrtol instead, in all the places it's changed.

Reply:
So the `_minimize_newtoncg` should be changed to xrtol at some point

Reply:
Sounds great, thanks for the feedback @andyfaff. I'll make the `xrtol` change for `_minimize_bfgs` now and the same change for `_minimize_newtoncg` in another commit (and any others that need it as well).